Sources/swiftformat/Run.swift

```diff
@@ -75,14 +75,18 @@ public func formatMain(path: String, isDebugMode: Bool) -> Int {
     // Important! We need to cast this to Syntax to avoid going directly into the specialized
     // version of visit(_: SourceFileSyntax), which will not run the pipeline properly.
     let formatted = pipeline.visit(file as Syntax)
-    let printer = PrettyPrinter(
-      configuration: context.configuration,
-      node: formatted,
-      isDebugMode: isDebugMode
-    )
-    print(printer.prettyPrint(), terminator: "")
-//    let output = url.deletingPathExtension().appendingPathExtension("formatted.swift")
-//    try formatted.description.write(to: output, atomically: true, encoding: .utf8)
+
+    if prettyPrint {
+      let printer = PrettyPrinter(
+        configuration: context.configuration,
+        node: formatted,
+        isDebugMode: isDebugMode
+      )
+      print(printer.prettyPrint(), terminator: "")
```

Comment:
If pretty printing is enabled we just write to stdout, but if we're not pretty printing, we get the old behavior of writing it out to `foo.formatted.swift`. Should we update the pretty-print part of the conditional to also write it out there?

Reply:
I'm not sure. For development purposes it is certainly easier to have it go to stdout.

Reply:
Long-term, we're going to need to audit how we produce the output anyway. Generating `foo.formatted.swift` is kind of clunky, but makes sense (1) while we're developing it so that we don't accidentally blow away someone's source by doing it in-place, and (2) lets us support passing multiple files in a single invocation.

I guess my big concern is that somebody running it with `-p` will get completely different behavior than if they run it without that flag, and that would be confusing. I think for that reason, we should unify them until we decide on a better approach across the board. WDYT?

Reply:
I changed the default behavior to print to stdout as well.